CHANGELOG.md

```diff
@@ -6,13 +6,74 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-### Changed
+## [0.16.0] - 2024-02-08
+### Added
+- Implemented the UARCH reset feature
+- Added soft yield runtime configuration using hints of the SRAIW instruction
+- Added shadow uarch state
+- Embedded the uarch RAM binary
+- Added instructions on how to build `libcartesi.a` in standalone
+- Supported compiling `libcartesi_jsonrpc.a` static library
+- Added file mapping and terminal support for Windows
+- Added the possibility to compile `libcartesi` without `mmap()`
+- Supported compiling `libcartesi` as a static library
+- Added memory range introspection
+- Introduced `-it` option for colored and bigger interactive terminal
+- Replaced MTD drives with PMEM drives
+- Initialized random entropy from the device tree
+- Added root hashes to `catalog.json`
+- Replaced proof with `sibling_hashes`
 
-- Changed stored machine configs from protobuf to JSON
-- Build device tree automatically into machine's ROM, eliminating the need for a ROM image
+### Changed
+- Built device tree automatically into the machine's ROM, eliminating the need for a ROM image
 - Enabled rollup by default
 - Updated libboost version to 1.81
+- Changed stored machine configs from protobuf to JSON
 - Removed remote-cartesi-machine-proxy
+- Made `libcartesi.a` self-contained again
+- Moved uarch generated files to the uarch source directory
+- Organized all tests under the tests directory
+- Removed emulator patch version from the device tree
+- Used new rootfs provided by the tools repository
+- Used static libraries when compiling executables
+- Made it possible to compile `libcartesi` without threading support
+- Reimplemented a portable OS time function
+- Allowed compiling `libcartesi` without GRPC or Lua installed
+- Didn't expose internal symbols in the public C API
+- Allowed cross-compilation with `TARGET_OS` make flag
+- Removed `std::filesystem` use from `libcartesi` (unsupported by WASI)
+- Made `termios` non-mandatory in `libcartesi` (unsupported by WASI)
+- Reordered `LDFLAGS` when linking to respect extra flags when packaging
+- Adjusted bootargs and renamed bootargs command line option
+- Introduced machine init and entrypoint config and options
+- Removed dump_pmas option
+- Removed Lua `md5` dependency by using keccak instead
+- Simplified machine hash computation in tests
+- Renamed ROM to DTB
+- Bumped kernel and rootfs versions
+- Replaced crypto++ with tiny sha3 to compute Keccak hash
+- Replaced boost filesystem with std filesystem
+- Updated mongoose to 7.12 and refactored how it's bundled
+- Bundled nlohmann-json into third-party
+- Replaced libb64 with a custom base64 implementation
+- Removed the unused `--json-steps` option
+
```

Comment:
I would few more lines here:
- Added support for compiling libcartesi to WebAssembly - because this releases did https://github.com/cartesi/machine-emulator/issues/59 and it even has a wiki page now

Also review if all changes that could impact Sunodo are listed here, I listed them in https://github.com/sunodo/sunodo/issues/300, I think most are.

Reply:
Please take a look